src/components/SearchPage/Pagination/Pagination.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react";
```

Comment:
Same situation as before, this logic below should be in container, and component should just receive data to render